app/shared/structures/content-type.structures.ts

```diff
@@ -0,0 +1,25 @@
+
+
+export interface NodePageContent {
+  title: string;
+  type: string;
+  slug: string;
+  description?: string;
+  relatedEntries?: NodePageContent[];
+}
+
+export interface VideoContent {
```

Comment:
yes, it was based on old layout , where you had eg blogpost and on the right few related blogposts